src/components/CategoryDropdown/CategoryDropdown.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
```

Comment:
Instead of making a very specific dropdown, is it possible to make this a reusable component that takes an addition array of dropdown selections so that you can use this for any number of selections?

Reply:
As per the ticket I was asked to hard code the categories for now, as we are planning to work with a fixed number of categories only, may be I can implement changes in the next sprint for this part if required as there are dependent tickets on this PR for this sprint.